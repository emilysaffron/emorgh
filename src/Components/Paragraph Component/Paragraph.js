import Grid from "@bbc/psammead-grid";
import Paragraph from "@bbc/psammead-paragraph";
import getGridGroups from "../../Helpers/GetGridGroups";

const TextBody = ({ text }) => {
  return (
    <Grid
      item
      startOffset={getGridGroups([2, 2, 2, 2, 2, 2])}
      columns={getGridGroups([5, 5, 5, 5, 5, 5])}
    >
      <Paragraph>{text}</Paragraph>
    </Grid>
  );
};
export default TextBody;
