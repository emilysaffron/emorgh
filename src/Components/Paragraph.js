import Grid from "@bbc/psammead-grid";
import Paragraph from "@bbc/psammead-paragraph";

const TextBody = ({ text }) => {
  return (
    <Grid
      item
      startOffset={{
        group0: 2,
        group1: 2,
        group2: 2,
        group3: 2,
        group4: 2,
        group5: 2,
      }}
      columns={{
        group0: 5,
        group1: 5,
        group2: 5,
        group3: 5,
        group4: 5,
        group5: 5,
      }}
    >
      <Paragraph>{text}</Paragraph>
    </Grid>
  );
};
export default TextBody;
