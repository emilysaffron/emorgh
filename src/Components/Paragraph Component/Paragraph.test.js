import Paragraph from "./Paragraph";
import renderer from "react-test-renderer";

it("Paragraph Snapshot", () => {
  const tree = renderer.create(<Paragraph />).toJSON();
  expect(tree).toMatchSnapshot();
});
;
