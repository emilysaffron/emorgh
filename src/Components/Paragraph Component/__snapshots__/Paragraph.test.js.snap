// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Paragraph Snapshot 1`] = `
<div
  className="css-19yvp8i-GridComponent e57qer20"
  dir="ltr"
>
  <p
    className="css-dv6jpn-Paragraph e1c0huex0"
  />
</div>
`;
